table.go

```diff
@@ -850,6 +932,55 @@ func (t *TableBlock) abort(commit func(), granule *Granule) {
 	}
 }
 
+func (block *TableBlock) Serialize() ([]byte, error) {
```

Comment:
the linter wants this variable to be named `t` as that's what it's called everywhere else